roles/openshift_imagestreamtags/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Ensure openshift project exists
+  oc_project:
+    name: openshift
+    state: present
+    node_selector:
+    - ""
+
+- name: Tag image into an image stream in the openshift namespace
+  command: >
+    {{ openshift_client_binary }} tag --source=docker {{docker_image_url}} --config={{ openshift.common.config_base }}/master/admin.kubeconfig openshift/{{ openshift_ist }}
```

Comment:
docker_image_url is not a great name for a variable.

We should probably create a list/dict so users can pass in more than one without have to adjust their inventory between runs.
```yaml
openshift_image_stream_tags:
  some_ist: source_url
  some_other_ist: source2_url
```

Reply:
update pushed - thanks @michaelgugino